backends/backends.c

```diff
@@ -246,6 +246,207 @@ static void backends_main_cleanup(void *ptr) {
     static_thread->enabled = NETDATA_MAIN_THREAD_EXITED;
 }
 
+/**
+ * Set Kinesis variables
+ *
+ * Set the variables necessaries to work with this specific backend.
+ *
+ * @param default_port  the default port of the backend
+ * @param brc function called to check the result.
+ * @param brf function called to format the msessage to the backend
+ * @param type the backend string selector.
+ */
+void backend_set_kinesis_variables(int *default_port
+        ,backend_response_checker_t brc
+        ,backend_request_formatter_t brf
+        ) {
```

Comment:
Please do better formatting for functions. Something like [this](https://github.com/netdata/netdata/blob/master/database/engine/rrdengine.c#L133).

Reply:
It was also changed for the next commit.